app/assets/stylesheets/earthworks.css

```diff
@@ -317,3 +317,29 @@ label.toggle-bookmark {
   --bs-link-color-rgb: var(--stanford-digital-blue-dark-rgb);
   text-decoration: underline;
 }
+
+/* Override mobile view for facets section in Blacklight */
+.facets {
+  --bl-facets-smallish-border: none;
+  --bl-facets-smallish-padding: 0;
+}
+
+/* To help vertically align on mobile */
+@media (max-width: 991.98px) {
+  .facets-heading {
+    margin-bottom: 0;
+  }
+}
+
+/* Hide/show button on mobile */
+.facets-header .navbar-toggler {
+  --bl-facets-smallish-border: none;
+  --bs-navbar-toggler-border-color: var(--stanford-digital-blue);
+  --bs-navbar-color: var(--stanford-digital-blue);
+  color: var(--stanford-digital-blue);
+}
+
+.facets-header .navbar-toggler:hover {
+  color: var(--stanford-digital-blue-dark);
```

Comment:
We should make this settable in blacklight: https://github.com/projectblacklight/blacklight/blob/69373f202753f04ec6aca179fdf8fa01248a9edf/app/assets/stylesheets/blacklight/_facets.scss#L22-L25

Reply:
Do you mean, for example, make a variable specifically for the hover color in Blacklight, so we can just override that  variable value?

Reply:
Exactly that.